readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://<monikaprochniewicz>.github.io/layout_search-bar-airbnb/)
```

Comment:
Link is broken, should be https://monikaprochniewicz.github.io/layout_search-bar-airbnb/ without <>